src/plugins/orangehrmDashboardPlugin/config/routes.yaml

```
@@ -26,6 +26,14 @@ apiv2_dashboard_employees_by_location:
   defaults:
     _api: OrangeHRM\Dashboard\Api\EmployeeDistributionByLocationAPI
 
+apiv2_dashboard_employee_time_at_work:
+  path: /api/v2/dashboard/employees/{empNumber}/time-at-work
```

```suggestion
  path: /api/v2/dashboard/employees/time-at-work
```
will move `empNumber` to query param

---

done